pt-BR/editor/ui/chrome/composer/editorOverlay.dtd

```diff
@@ -570,31 +570,31 @@ entities below were automatically added by l10nmerge
 <!ENTITY Heading6Abbr.label "H6">
 <!ENTITY fileprint.accesskey "p">
 <!ENTITY fontHelveticaFont.label "Helvetica, Arial">
-<!ENTITY exitCmd.label "Quit">
+<!ENTITY exitCmd.label "Sair">
 <!ENTITY insertisindex.accesskey "x">
 <!ENTITY heading2.accesskey "2">
 <!ENTITY toolsetfocus.accesskey "f">
 <!ENTITY inserthline.accesskey "o">
 <!ENTITY decreaseindentkb "-">
 <!ENTITY insertinputimage.accesskey "I">
-<!ENTITY pasteTextCmd.label "Text">
+<!ENTITY pasteTextCmd.label "Texto">
 <!ENTITY newBlankPage.accesskey "P" >
-<!ENTITY textColorCaption.label "Text Color">
-<!ENTITY insertFormMenu.label "Form">
+<!ENTITY textColorCaption.label "Cor do texto">
+<!ENTITY insertFormMenu.label "De">
 <!ENTITY tableinsertmenu.accesskey "i">
-<!ENTITY editStripQuotesCmd.label "Strip Quotes">
+<!ENTITY editStripQuotesCmd.label "Remover citações">
 <!ENTITY insertformmenu.accesskey "F">
 <!ENTITY editfind.accesskey "f">
-<!ENTITY pasteRowsCmd.label "Rows">
+<!ENTITY pasteRowsCmd.label "Linhas">
 <!ENTITY formatparagraphmenu.accesskey "P">
 <!ENTITY editpastelink.accesskey "l">
 <!ENTITY pasteImage.accesskey "i">
 <!ENTITY tablefix.accesskey "f">
 <!ENTITY tablesplitcell.accesskey "c">
 <!ENTITY fileopen.keybinding "o">
 <!ENTITY increaseFontSize.keybinding "+">
-<!ENTITY insertInputTagCmd.label "Form Field…">
-<!ENTITY viewParagraphMarks.label "Paragraph Marks">
+<!ENTITY insertInputTagCmd.label "Campo de…">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY insertInputTagCmd.label "Campo de Formulário...''>